### PR TITLE
AT_16.01_011 | Multibranch Pipeline>Build Configuration

### PR DESCRIPTION
**Pull request by Gulya111**
https://trello.com/c/cmdAdKim/1366-tc1601011-multibranch-pipelineverify-visibility-of-fields-namesbuild-configuration

### CODE REVIEW

**Comment by S-Lanchik** [comment]
Why do you use so many assertions?

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js [hunk 1]
```diff
@@ -41,4 +42,22 @@ describe('Multibranch Pipeline Configuration', function () {
...
+            .should('contain', multibranchPipline.configurationsFields.byJenkinsfile)
+        cy.get('.jenkins-form-item > .jenkins-form-label')
+            .should('contain', multibranchPipline.configurationsFields.scriptPath)
+        cy.get('a[title="Help for feature: Script Path"]')
+            .realHover()
+            .should('be.visible')
+        cy.get('input[name="_.scriptPath"]')
+            .should('be.visible')
+            .clear().type(scriptPathName)
+        cy.get('body').click({ force: true });
```
Comment (DariaMas7) [inline cypress/e2e/multiBranchPipelineConfiguration.cy.js:61]: The last method in our test cases shouldn't be just click()
You should make some assertion regarding the name that you typed, that its really saved

**Review by DariaMas7 — COMMENTED** [review]
Also, there is a lot of assertion on different topics you should narrow the subject of your AT

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js (hunk 1 above)
Comment (Gulya111) [reply cypress/e2e/multiBranchPipelineConfiguration.cy.js]: I narrowed the subject to only visibility of field's names Build Configuration

**Review by RimmaSalikh — APPROVED** [review]

**Review by larysareznikova — APPROVED** [review]

**Pull request merged**